presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -1467,6 +1467,39 @@ public void testChiSquaredCdf()
         assertInvalidFunction("chi_squared_cdf(3, -10)", "value must non-negative");
     }
 
+    @Test
+    public void testInverseGammaCdf()
+    {
+        assertFunction("inverse_gamma_cdf(3, 3.6, 0.0)", DOUBLE, 0.0);
+        assertFunction("round(inverse_gamma_cdf(3, 4, 0.99), 3)", DOUBLE, 33.624);
+        assertFunction("round(inverse_gamma_cdf(3, 4, 0.50), 3)", DOUBLE, 10.696);
+        // Gamma with shape 10k/2 and scale 2 is chisquare with df=10k, which is approximatly Normal with mu=10k
+        // Hence, we expect that the quantile will be close to 10k, as we indeed get:
+        assertFunction("round(inverse_gamma_cdf(10000.0/2, 2.0, 0.50), 3)", DOUBLE, 9999.333);
+
+        assertInvalidFunction("inverse_gamma_cdf(0, 3, 0.5)", "shape must be greater than 0");
+        assertInvalidFunction("inverse_gamma_cdf(3, 0, 0.5)", "scale must be greater than 0");
+        assertInvalidFunction("inverse_gamma_cdf(3, 5, -0.1)", "p must be in the interval [0, 1]");
+        assertInvalidFunction("inverse_gamma_cdf(3, 5, 1.1)", "p must be in the interval [0, 1]");
+    }
+
+    @Test
+    public void testGammaCdf()
+            throws Exception
```

Comment:
`Exception 'java.lang.Exception' is never thrown in the method `, remove this part

Reply:
Thanks, done.
(might be worth then to remove it from other probability functions then. This was never mentioned before. But if to remove it from other functions in the file, I propose to do it in another diff)